src/components/TodoList.js

```diff
@@ -0,0 +1,19 @@
+import { FaTrash } from 'react-icons/fa';
+
+const TodoList = ({ title, isChecked, handleDelete, handleComplete }) => {
+  return (
+    <div className='flex justify-between gap-5 items-center mt-5 border-b-2 border-gray-100 pb-2'>
+      <div className='flex gap-3 items-baseline'>
+        <input type='checkbox' id='todo' onClick={handleComplete} />
```

Comment:
It is not recommended to mix controlled and uncontrolled input. You can add the `checked` attribute on the input to make it fully controlled.
```suggestion
        <input type='checkbox' id='todo' checked={isChecked} onClick={handleComplete} />
```

Reply:
I think because my input checkbox doesn’t have the onChange handler, there is no need to give it a checked attribute.